src/components/cluster/ClusterForm.tsx

```diff
@@ -132,6 +132,7 @@ export default function ClusterForm({
     isClusterDetails,
     toggleClusterDetails,
     isVirtualCluster,
+    isProd
```

Comment:
Couple of suggestions:
- isProd should have boolean type
- We can start adding types like: any & { isProd: boolean } or record